### DISCUSSION TITLE
Actividad 0 - Robledo, Nuria

### DISCUSSION THREAD

**Discussion by nuriarobledo** [frlp-utn-ingsoft/catedra#44] [Show and tell]
https://github.com/nuriarobledo/ingSoftware-Actividad0

**Comment by RodrigoJacznik** [comment]
Hola, los commits y sus mensajes son correctos pero no pude correr la app. Me tira un error en el template de `detail`. A vos te funciona?

**Reply by nuriarobledo** [reply]
Hola, hice una corrección, era el index.html. Ya lo solucione !

**Reply by ivanduva** [reply]
Perfecto! La actividad quedó aprobada
